Resources/public/js/views/services/ez-serversideviewservice.js

```diff
@@ -118,10 +118,12 @@ YUI.add('ez-serversideviewservice', function (Y) {
          */
         _parseResponse: function (response) {
             var frag = Y.Node.create(response.responseText),
-                html, title;
+                that = this,
+                html, title, notifications;
 
             html = frag.one('[data-name="html"]');
             title = frag.one('[data-name="title"]');
+            notifications = frag.one('[data-name="notification"]');
```

Comment:
you don't really need this Node actually, so instead of the `if` condition below, you can write something like:

```js
frag.all('[data-name="notification"] li').each(function (notificationNode) {
    that._responseNotify(notificationNode);
});